extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/PanacheMongoEntityBase.java

```diff
@@ -864,6 +864,49 @@ public static void persistOrUpdate(Object firstEntity, Object... entities) {
         MongoOperations.persistOrUpdate(firstEntity, entities);
     }
 
+    /**
+     * Update all entities of this type matching the given query, with mandatory indexed parameters.
```

Comment:
I'm not sure what the `mandatory` here means. It should be removed.

Reply:
Below (for repository) it's `optional`…

Reply:
It's copied from the JavaDoc of Hibernate with Panache where `mandatory` is used. I think it's wrong as we should be able to use it without any parameters : `Person.update("set person.status = 'Dead' where person.status='Alive');`

I check the code of the Hibernate with Panache implementation and no parameters are allowed at code level, I'll update the documentation with `optional` here also.

Reply:
Yeah, I've no idea how this could have slipped through. Thanks for fixing it there too.